src/test/splitting/splitting/find_task_df_test.py

```diff
@@ -1,12 +1,12 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
+import pytest
+import pandas as pd
```

Comment:
Pytest and pandas are not the standard libraries, so please, move these imports into a separated second group